Source/RxComponents/LMCAnalyticResponseFunction.cc

```diff
@@ -0,0 +1,64 @@
+/*
+ * LMCAnalyticResponseFunction.cc
+ *
+ *  Created on: Jul6, 2022
+ *      Author: pslocum
+ */
+
+#include "LMCAnalyticResponseFunction.hh"
+#include <iostream>
+#include "logger.hh"
+
+namespace locust
+{
+
+	LOGGER( lmclog, "AnalyticResponseFunction" );
+
+
+    AnalyticResponseFunction::AnalyticResponseFunction():
+	    fGeneratingTF( false ),
+		fInitialFreq( 0. ),
+		fTFarray(0 )
+    {}
+    AnalyticResponseFunction::~AnalyticResponseFunction() {}
+
+
+    bool AnalyticResponseFunction::Configure( const scarab::param_node& aParam )
+    {
+        return true;
```

Comment:
Is this just a placeholder for the time being and will be filled in later, or is there really nothing meaningful to configure here from the `aParam`?

Reply:
Right, it is a placeholder to be filled in as needed.  At the moment we are doing the configuration in its derived classes DampedHarmonicOscillator and EquivalentCircuit.